### PR TITLE
chore: remove tailwind base classes for border radius

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Set the defaults to undefined. Once undefined, it becomes excluded from the classes list. Proven by the first commit which had a failing build for using `@apply` on an unknown class.
- Create a reusable function to remove a whole object.
- Start removing undefined from rounded.
- This also removed `rounded` as it apparently has a value of `4px` which is not intuitive with our current setup. With that, I introduced a rounded-4

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-80 #done

### CODE REVIEW

File: packages/shared/tailwind.config.js [hunk 1]
```diff
@@ -186,24 +201,30 @@ module.exports = {
...
-        18: '1.125rem',
-        22: '1.375rem',
-        24: '1.5rem',
-        26: '1.625rem',
-        32: '2rem',
-        40: '2.5rem',
-        48: '3rem',
-        50: '3.125rem',
-      },
+      borderRadius: removeDefaults(
```
Comment (rebelchris) [inline packages/shared/tailwind.config.js:204]: We should probably not use extend, but overwrite then you don't need to remove defaults :)

File: packages/shared/tailwind.config.js [hunk 2]
```diff
@@ -186,24 +201,30 @@ module.exports = {
...
+        defaultTheme.borderRadius,
+        ['full', 'none'],
+        {
+          half: '50%',
```
Comment (rebelchris) [inline packages/shared/tailwind.config.js:208]: Probably better to stick to 1/2 naming as it's common for tailwind

**Review by rebelchris — CHANGES_REQUESTED** [review]
Minor suggestion we discussed, but might have gotten lost in translation :)

File: packages/shared/tailwind.config.js (hunk 1 above)
Comment (sshanzel) [reply packages/shared/tailwind.config.js:204]: I see, I can't seem to find the docs for overwrite, do you maybe know a sample of how to do it?

File: packages/shared/tailwind.config.js (hunk 2 above)
Comment (sshanzel) [reply packages/shared/tailwind.config.js:208]: Yep. Agree with that.

File: packages/shared/tailwind.config.js (hunk 1 above)
Comment (sshanzel) [reply packages/shared/tailwind.config.js:204]: Ahh, I see it now. It is under the `extend` object.

Comment (rebelchris) [reply packages/shared/tailwind.config.js:204]: Yeah: https://tailwindcss.com/docs/theme#overriding-the-default-theme

**Review by rebelchris — APPROVED** [review]
Looks great, thanks Lee!

**Review by omBratteng — APPROVED** [review]

**Pull request merged**